Docs/source/usage/workflows/generate_lookup_tables_with_tools.rst

```diff
@@ -0,0 +1,44 @@
+.. _generate-lookup-tables-with-tools:
+
+Generate QED lookup tables using the standalone tool
+======================
+We provide tools to generate and convert into a human-readable format the QED lookup tables.
```

Comment:
I think Sphinx needs these to be aligned:
```suggestion
Generate QED lookup tables using the standalone tool
====================================================

We provide tools to generate and convert into a human-readable format the QED lookup tables.
```

Reply:
Thanks!